packages/shared/src/utils/contract/contract.ts

```diff
@@ -154,3 +154,9 @@ export const getContractUpdateConfig = ({ contract_update, limit_order }: TGetCo
 export const shouldShowExpiration = (symbol: string) => /^cry/.test(symbol);
 
 export const shouldShowCancellation = (symbol: string) => !/^(cry|CRASH|BOOM|stpRNG|WLD|JD)/.test(symbol);
+
+export const getSubType = (type: string) => {
+    const sub_type: string[] = type.toLowerCase().replace('turbos', '').split('');
+    sub_type[0] = sub_type[0].toUpperCase();
+    return `${sub_type.join('')}`;
```

Comment:
```suggestion
    return `${subtype.join('')}`;
```
@kate-deriv i think subtype without underscore is totally fine)